public/apidocs/swagger_v3.json

```diff
@@ -2593,6 +2652,12 @@
           }
         }
       },
+      "variantsECDC": {
+        "type": "array",
+        "items": {
+          "type": "string"
```

Comment:
This doesn't look right. See how other endpoints do this where they return an array of type `x` and then go on to declare `x` as the object structure of the element. Like
```json
"updated": dateTime,
"country": string,
"yearWeek": string...
```